e2e/container/create_test.go

```diff
@@ -32,4 +32,50 @@ func TestCreateWithContentTrust(t *testing.T) {
 	})
 }
 
-// FIXME(vdemeester) TestTrustedCreateFromBadTrustServer needs to be backport too (see https://github.com/moby/moby/pull/36515/files#diff-4b1e56bb77ac16f2ccf956fc24cf0a82L331)
+func TestTrustedCreateFromUnreachableTrustServer(t *testing.T) {
+	dir := fixtures.SetupConfigFile(t)
+	defer dir.Remove()
+	image := fixtures.CreateMaskedTrustedRemoteImage(t, registryPrefix, "trust-create", "latest")
+
+	result := icmd.RunCmd(
+		icmd.Command("docker", "create", image),
+		fixtures.WithConfig(dir.Path()),
+		fixtures.WithTrust,
+		fixtures.WithNotaryServer("https://invalidnotaryserver"),
+	)
+	result.Assert(t, icmd.Expected{
+		ExitCode: 1,
+		Err:      "error contacting notary server",
+	})
+}
+
+func TestTrustedCreateFromBadTrustServer(t *testing.T) {
+	dir := fixtures.SetupConfigFile(t)
+	defer dir.Remove()
+
+	evilNotaryDir := fixtures.SetupConfigFileWithNotaryURL(t, fixtures.EvilNotaryURL)
+	defer evilNotaryDir.Remove()
+
+	evilImageName := "registry:5000/evil-alpine:latest"
+	icmd.RunCmd(icmd.Command("docker", "pull", fixtures.AlpineImage)).Assert(t, icmd.Success)
+	icmd.RunCmd(icmd.Command("docker", "tag", fixtures.AlpineImage, evilImageName),
+		fixtures.WithConfig(evilNotaryDir.Path()),
+	).Assert(t, icmd.Success)
+	icmd.RunCmd(icmd.Command("docker", "image", "push", evilImageName),
+		fixtures.WithConfig(evilNotaryDir.Path()),
+		fixtures.WithPassphrase("root_password", "repo_password"),
+		fixtures.WithTrust,
+		fixtures.WithNotaryServer(fixtures.EvilNotaryURL),
+	).Assert(t, icmd.Success)
+
+	icmd.RunCmd(icmd.Command("docker", "image", "rm", evilImageName)).Assert(t, icmd.Success)
+
+	icmd.RunCmd(icmd.Command("docker", "create", evilImageName),
+		fixtures.WithConfig(dir.Path()),
+		fixtures.WithTrust,
+		fixtures.WithNotary,
+	).Assert(t, icmd.Expected{
+		ExitCode: 1,
+		Err:      "remote trust data does not exist for registry:5000/evil-alpine",
```

Comment:
When I look at the [removed test](https://github.com/moby/moby/pull/36515/files#diff-4b1e56bb77ac16f2ccf956fc24cf0a82L360) on `moby/moby` I can't see the same result as yours.
Can you add some context in the PR comment to explain why you expect this result, and maybe in the test too?

Reply:
Well, updating the WithNotary ends up with the command exitCode equals to 0. There may be another problem with the configuration.

Reply:
Finally, after looking to the previous test, I was actually missing the first part. I updated the test and report comments from the previous test.

Reply:
I can't see the comments from `moby/moby` test, did you push them?

Reply:
Sorry for this, I just pushed the update.